demos/nx-next-monorepo-demo/apps/demo-monorepo-e2e/project.json

```diff
@@ -1,4 +1,5 @@
 {
+  "name": "demo-monorepo-e2e",
```

Comment:
The project.json files got generated and workspace.json got deleted when updating nx dependencies as part of the update process.